central/globaldb/metrics/metrics.go

```diff
@@ -201,6 +204,27 @@ var (
 		Name:      "postgres_connected",
 		Help:      "flag indicating if central is connected to the Postgres Database. 0 NOT connected, 1 connected",
 	})
+
+	PostgresActiveConnections = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.CentralSubsystem.String(),
+		Name:      "postgres_active_db_connections",
+		Help:      "number of active connections to Postgres by database name",
+	}, []string{"database"})
+
+	PostgresTotalConnections = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
I would suggest that we don't split these, and instead add it as a label so it would be like state: idle, active. This lets you visualize everything more easily

Reply:
and also takes into accounts other states besides active and not active